### ISSUE TITLE
Harvest Tab

### ISSUE THREAD

**Issue by gfrc11752** [Huh/PopR_SDGFP#111]
Any chance we can get tag numbers by tag type per unit as a query option in the harvest tab?

**Comment by Huh** [comment]
Yes.  If I understand we want to return the number of tags instead of harvest. For the moment, does the IPM tab give the information you want?
